env.dev.pg.base

```diff
@@ -130,14 +117,13 @@ DATASET_DOWNLOAD_SERVER_PORT=80
 
 # Image Versions
 # Handled by VDI_SCHEMA_SUFFIX and compose default
```

Comment:
Can remove this comment line "Handled by...", as it refers to the commented line you removed.